src/lucky/maximum_request_size_handler.cr

```diff
@@ -0,0 +1,40 @@
+class Lucky::MaximumRequestSizeHandler
+  include HTTP::Handler
+
+  Habitat.create do
+    setting enabled : Bool = false
+    setting max_size : Int32 = 1_048_576 # 1MB
```

Comment:
:+1:

Reply:
actually... if you wanted this to be say 10GB (video uploads), would this need to be Int64?